docs/user/reference/secret.md

```diff
@@ -120,6 +120,23 @@ The `vault` backend is the only one that supports specific configuration keys. T
 
 A minimum configuration must include the `endpoint` and `token`. However, just that would not be insecure, as it wouldn't establish an encrypted TLS connection to Vault. For production you should configure your Vault securely, following recommendations in the upstream Vault documentation.
 
+The `kubernetes` backend supports the following configuration keys:
+
+|||
+|---|---|
+| `ca-cert` | A PEM-encoded CA certificate. This comes from base64 decoding the `certificate-authority-data` value in the kube config file.|
+| `ca-certs` | A list of A PEM-encoded CA certificates (if a cert chain is used).|
```

Comment:
```suggestion
| `ca-certs` | A list of PEM-encoded CA certificates (if a cert chain is used).|
```

Reply:
(Not sure if I understood this right, but it looked liked a potential typo.)

Reply:
yes, thanks for spotting it